projects/plugins/jetpack/extensions/blocks/ai-assistant/lib/prompt/index.ts

```diff
@@ -130,21 +175,21 @@ export const buildPromptTemplate = ( {
 	relevantContent = null,
 	isContentGenerated = false,
 	fullContent = null,
-	isGeneratingTitle = false,
+	isTitlePrompt = false,
 }: {
 	rules?: Array< string >;
 	request?: string;
 	relevantContent?: string;
 	isContentGenerated?: boolean;
 	fullContent?: string;
-	isGeneratingTitle?: boolean;
+	isTitlePrompt?: boolean;
 } ): Array< PromptItemProps > => {
 	if ( ! request && ! relevantContent ) {
 		throw new Error( 'You must provide either a request or content' );
 	}
 
 	// Add initial system prompt.
-	const messages = [ getInitialSystemPrompt( { rules } ) ];
+	const messages = [ getInitialSystemPrompt() ];
```

Comment:
Why did you remove the ability to extend the system rules?

Reply:
Because I separated the rules in another message, so they were moved [here](https://github.com/Automattic/jetpack/pull/31421/files#diff-49fc00a7ab90da5972cc0277af7a647b8059ae846c1952515f7a1cad8be20652R215).